app/controllers/admin/admin_users_controller.rb

```diff
@@ -133,7 +133,7 @@ def confirm_delete_user_creations
     @works = @user.works.paginate(page: params[:works_page])
     @comments = @user.comments.paginate(page: params[:comments_page])
     @bookmarks = @user.bookmarks
-    @collections = @user.collections
+    @collections = @user.collections.to_a.delete_if { |collection| !(collection.all_owners - @user.pseuds).empty? }
```

Comment:
Might be worth adding an explicit `def sole_owned_collections` to `user.rb`?

Reply:
I updated destroy_user_creations, thank you so much for that catch. All is working as expected.

I can add it here, but not completely sure what it would do? Would this be creating a universal variable?

Reply:
I meant adding a method in the User class like:
```rb
class User < ApplicationRecord
# ...
  def sole_owned_collections
    collections.to_a.delete_if { |collection| !(collection.all_owners - @user.pseuds).empty? }
  end
```
So we can just call `user.sole_owned_collections` in relevant locations instead of copy-pasting the full line every time.

Reply:
Maybe using [@user.owned_collections](https://github.com/otwcode/otwarchive/blob/54e5f2640a129acd3a1e593362c62bc0a9b01957/app/models/user.rb#L102) and adding a new filter on it would be more efficient? 

Also, we should update [this line](https://github.com/otwcode/otwarchive/blob/54e5f2640a129acd3a1e593362c62bc0a9b01957/app/controllers/users_controller.rb#L180) as well.

Reply:
Yes thank you, I remembered that one, its where I got the code from! I added a def sole_owned_collections, but can change to the filter if ya'll think that's best.